# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cookie-store/__dir__.ini
# Commit: d55b6525d7b2
# Full Hash: d55b6525d7b24d90ac7e1b092e4fef698cc244fe
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 1 - CookieStore API - WebIDL, r=smaug,edgul
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215144
# ==============================================================================

diff -r 26f637a20dc1 -r d55b6525d7b2 testing/web-platform/meta/cookie-store/__dir__.ini
--- a/testing/web-platform/meta/cookie-store/__dir__.ini	Thu Sep 12 20:45:28 2024 +0300
+++ b/testing/web-platform/meta/cookie-store/__dir__.ini	Thu Sep 12 16:48:18 2024 +0000
@@ -1,3 +1,4 @@
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1475599
+prefs: [dom.cookieStore.enabled:true]
 implementation-status: backlog
 lsan-allowed: [Alloc, Malloc, Then, mozilla::BasePrincipal::CreateContentPrincipal, mozilla::dom::DocGroup::Create, mozilla::dom::ServiceWorkerJobQueue::RunJob, mozilla::dom::ServiceWorkerManager::Unregister, mozilla::dom::ServiceWorkerRegistrationMainThread::Unregister, mozilla::dom::UnregisterCallback::UnregisterCallback, mozilla::net::nsStandardURL::TemplatedMutator, operator]